src/Form/Checkout/CheckoutPaymentType.php

```diff
@@ -42,6 +42,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ])
                 ->add('installments', HiddenType::class, [
```

Comment:
maybe add a comment here explaining the `installments` concept for the payment form? it is also specific to mercadopago ?

Reply:
Installments is a general payment concept, it means "pay in 3 times" for example.